lib/default_cpolicy.json

```diff
@@ -0,0 +1,92 @@
+{
+	"rules": 
+	[
+		{
+			"name": "pln-rule-deduction",
+			"file_path": "reasoning/RuleEngine/rules/pln/deduction.scm",
+			"priority": 1,
+			"mutex": 
+			[
+				
+			],
+
+			"category": "PLN"
+		},
+
+		{
+			"name": "pln-rule-modus-ponens",
+			"file_path": "reasoning/RuleEngine/rules/pln/modus-ponens.scm",
+			"priority": 1,
+			"mutex": 
+			[
+				
+			],
+
+			"category": "PLN"
+		},
```

Comment:
I mean, just to be clear, instead of writing json, you could just have written this:
```
(EvaluationLink
   (PredicateNode "default_cpolicy")
   (ListLink
       (ConceptNode "pln-rule-modus-ponens")
       (ConceptNode "priority:1")
       (ConcpetNode "mutex")
       (ConceptNode "category:PLN")
   ))
```
that way, you would not need a json file, nor a json file reader.